integreat_cms/cms/models/pois/poi.py

```diff
@@ -93,6 +94,19 @@ class POI(AbstractContentModel):
         default=get_default_opening_hours,
         verbose_name=_("opening hours"),
     )
+    organization = models.ForeignKey(
+        Organization,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="organization",
+        verbose_name=_("organization"),
+    )
+    barrier_free = models.BooleanField(
+        default=False,
```

Comment:
```suggestion
        null=True,
        default=None,
```
Hmm, what do you think about storing 3 states here - `True` for barrier-free, `False` for "known not to be barrier free" and `None` when there is no info about it? Otherwise, we cannot differentiate between "not barrier free" and "unknown"... Not sure whether this is required from a content perspective though...
@osmers do you have an opinion on that?

Reply:
Yeah, I think that would make sense bcs it might not always be known - just make sure you name it clearly (so that the Kommunen know which one if for unknown and which is for no access) in the form and there shouldn't be a problem :)